common/toggle/reports/pre-ally-1.json

```
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/v4.0-ci/demos/toggle/toggle-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of toggle",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "gc-features"
```

This CSS selector are not accurate.

```suggestion
            "@value": ".wb-toggle"
```